ui/src/fragments/Icons/Ethereum.svelte

```diff
@@ -1,11 +1,9 @@
 <svg
   xmlns="http://www.w3.org/2000/svg"
-  fill="none"
-  viewBox="0 0 24 24"
-  stoke-width="1"
-  stroke="currentColor"
-  ><path
-    fill="currentColor"
-    d="M11.944 17.97L4.58 13.62 11.943 24l7.37-10.38-7.372 4.35h.003zM12.056 0L4.69 12.223l7.365 4.354 7.365-4.35L12.056 0z"
-  /></svg
->
+  viewBox="0 0 50 50"
```

Comment:
It's better if we find icons with the same `viewBox` property if possible, otherwise it makes sizing them inconsistent with other icons.